#goal with x, y, theta
float32 x
float32 y
float32 theta
---
#result success or failure
bool goal_status
string error_message
---
#feedback of completion percentage
int32 completion_rate
